src/views/evaluation/Evaluation.js

```diff
@@ -1,15 +1,15 @@
-const NAV = function (option = {}){
+const NAV = function (option = {}, logo = {}){
```

Comment:
Any particular reason why you chose to add another parameter here, instead of having it be part of `option` (which I'd argue would be better named `options` - plural)?